ddcloud/resource_server.go

```diff
@@ -190,12 +191,13 @@ func resourceServer() *schema.Resource {
 				Default:     "",
 				Description: "The IP address of the server's secondary DNS server",
 			},
-			resourceKeyServerAutoStart: &schema.Schema{
-				Type:        schema.TypeBool,
+			resourceKeyServerPowerState: &schema.Schema{
```

Comment:
We may need to attach a function here to ensure that the property has a valid value. Here's an example:

https://github.com/DimensionDataResearch/dd-cloud-compute-terraform/blob/1b26fb22685a573aef7dec14da418e28c7c50f43/ddcloud/resource_ip_address_reservation.go#L46-L49

Reply:
Not a problem.  Running local tests now before committing